README.md

```diff
@@ -31,217 +49,213 @@ wiki
-    > bundle install
-
-6. Add the following block anywhere in the `_config.yml` file
-    ```yml
-    plugins:
-        - jekyll-remote-theme
-    ```
-
-7. Run the following command to build the website on your local machine. 
-    > bundle exec jekyll serve
-
-#### MacOS
-MacOS comes with a default system installation for Ruby. However, it's advisable to use a seperate version of Ruby installed using "homebrew" to avoid permission errors.
+    home-page content:
+    ![readme_homepage](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/technical/readme_homepage.png)
 
-1. Navigate to your system's root folder and install command-line-tools 
-    > xcode-select --install
+## Uploading Images
 
-    > export SDKROOT=$(xcrun --show-sdk-path)
+To add an image to any wiki page:
+1. Place the image file within the most appropriate subfolder under `docs/_assets/`.
+2. Name the image file with an intuitive name. Many existing wiki images are named `[topic]_[detail].png`, such as: `readme_nav-example.png`.
+3. Insert the image into your webpage with the appropriate markdown:
 
-2. Install Homebrew by following the instructions on the [Homebrew homepage](https://brew.sh/) to paste a specific command into your Terminal.
-
-3. Install Ruby using homebrew
-    > brew install ruby
-
-    > echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.bash_profile
-
-5. Install bundler and jekyll by running:
-    > gem install --user-install bundler jekyll
+```yml
+![image-file-name](https://raw.githubusercontent.com/NDCLab/wiki/main/image-file-path)
+```
 
-6. Navigate to your local `wiki` repository and initialize the repo.
-    > cd wiki
-    > bundle init 
+For example, the image that appears above (showing the wiki homepage) is called in this markdown file as:
 
-7. This will create a gemfile called `Gemfile`. Add the following to this newly created file: 
-    ```ruby
-    gem "just-the-docs"
-    gem "jekyll-remote-theme"
-    ```
-
-8. Install the newly listed gems using: 
-    > bundle install
+```yml
+![readme_homepage](https://raw.githubusercontent.com/NDCLab/wiki/main/docs/_assets/technical/readme_homepage.png)
+```
 
-9. Add the following block anywhere in the `_config.yml` file
-    ```yml
-    plugins:
-        - jekyll-remote-theme
-    ```
+This method for linking images has two key benefits:
+1. The image file is easily accessible within the wiki repo.
+2. Given that the `main` branch is the published branch, the image displays correctly upon publication.
 
-10. Run the following command to build the website on your local machine. 
-    > bundle exec jekyll serve
+However, this method does have one major disadvantage of which users should be aware: the image will **not** render correctly on their working branch until it is merged with `main`. As a result, users are encouraged to double-check all images after the merge with `main` to ensure they are displaying as intended.
```

Comment:
Is there any way around this to allow for users to be able to properly view the embedded images before the branch is merged into main? It seems like this is a fairly big issue, no?

Reply:
@F-said @jessb0t

Reply:
The only workarounds I have found are:

1. Code the path with the dev branch first, then switch that at PR.  (Which anyone can do if they want, of course.)
2. Use a relative path, which I just tested.  It renders properly on GitHub Preview...but is not rendered properly by GitHub pages.

But, to be honest, this hasn't felt like a major impediment to me and everyone who is working through Get with Git basically sees it on their very first GitHub activity in the lab.

Reply:
P.S. I did a bunch of hunting around GitHub and Stack Overflow, so this isn't based solely on my own testing.